src/sagas/filtersAside.js

```diff
@@ -18,7 +18,8 @@ const defaultStatParams    = {
   with_content_types: true,
   with_persons: false,
   with_original_languages: false,
-  with_media: false
+  with_media: false,
```

Comment:
What are those params for?

Reply:
what kind of filters needs counters. 
You mean add comment for describe API params?